### DISCUSSION TITLE
Impossibile utilizzare la app HA

### DISCUSSION THREAD

**Discussion by sky7176** [indomus/forum#2532] [HUB: Home Assistant]
Buon giorno,
da questa mattina, non riesco ad utilizzare la app.
Si apre ma se cerco di navigare della dash board non ottengo nulla.
Se provo ad aprire da brouser, la dashboard non funziona, ma riesco a navigare nei menù sulla sinistra.
Ho provato a riavviare il tutto, ma nulla.

Tutti i dati si aggiornano, i comandi funzionano e pure androi auto va

HA su RB4 aggiornato all'ultima release.

Avete suggerimenti da darmi per cntrollare ?

Grazie
Giuseppe

**Comment by sky7176** [comment]
Aggiornamento:
ho provato a creare una nuova plancia e sembra funzionare, nel senso che mi fa interagire.
Avete idea di come possa copiare le schede da una plancia ad un'altra?
Mi viene male al solo pensiero di dover rifare tutto.

**Reply by Scialla** [reply]
le schede le puoi spostare, copiare, eliminare da UI (tre pallini in alto, modifica) e metterle sulla plancia che ti pare

**Reply by sky7176** [reply]
Grazie per la risposta, ma il mio problema era che la plancia principale era ingestibile, non riuscivo a muovermi tra le varie tab e all'interno delle stesse.

Poi come è apparso è scomparso.
C'è un post aperto da un utente sul forum di home-assistant a cui è accaduta la medesima cosa ieri.

**Reply by Scialla** [reply]
Se si è risolta, meglio così... :)
